### ISSUE TITLE
can drink blood with vampirism

### ISSUE THREAD

**Issue by GlazedHamRiot** [adam9899/Issues#167]
I changed the key binds and looked everywhere else. whenever I press v on a "drinkable creature" it doesn't do anything but I know its working since it hurts "non-drinkable creatures". this makes it literally impossible to survive as a vampire.

**Comment by ByteTemplar** [comment]
I don't know if the book tells the mechanics, but I believe you need to have Vampire fangs item for you to successfully suck blood...

**Comment by KajuzKajus** [comment]
No, try to rebind the key, and hold it, not press it, I had the same problem.
